idunn/blocks/wikipedia.py

```diff
@@ -218,18 +237,21 @@ def get_summary(self, title, lang):
     @WikipediaBreaker.handle_requests_error
     def get_title_in_language(self, title, source_lang, dest_lang):
         url = self.API_PHP_BASE_PATTERN.format(lang=source_lang)
-        resp = self.session.get(
-            url=url,
-            params={
-                "action": "query",
-                "prop": "langlinks",
-                "titles": title,
-                "lllang": dest_lang,
-                "formatversion": 2,
-                "format": "json",
-            },
-            timeout=self.timeout,
-        )
+
+        with self._prom_tracker.log_request_duration("wiki_api"):
```

Comment:
shouldn't we have different handler for the title than the sumary ?

Reply:
Since both are requesting the wiki api, I thought a single handler was ok for both of them.
No ?

Reply:
it's strange to aggregate the metrics on those calls (than ban be quite different)

Reply:
yup, done